portugues/titanic/pular/boiar/voce/voce.md

```diff
@@ -5,3 +5,9 @@ Tristemente você não tinha internet e não pode fazer push do seu trabalho.
 Você é despedido por que não entregou o código.
 
 -Whitney Huston comeca a tocar-
+
+Você vai dar uma volta pela praia, e você vê algo que parece o seu portátil.
+
+Vendo mais de perto, é mesmo o seu portátil. Você leva para casa e como é um Thinkpad você consegue recuperar [seus dados](https://www.youtube.com/watch?v=3KXx-Kys-Fk)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.